bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -351,16 +353,14 @@ def encode_forward_causallm(self, token_batch, tokenizer=None, encode_len=bitten
                 logits_std (:obj:`torch.FloatTensor`):
                     The nucleus's logit outputs as a torch tensor of shape [batch_size, sequence_len, __vocab_size__]
         """
-
         tokens = self.token_remap(token_batch, std_tokenizer=tokenizer, return_offsets_mapping=True)  # remap to server tokenizer
 
         def _forward(_model_output=model_output):
             if _model_output is None:
                 # transformer models like gerpt2 typically perform worse with left-side attention mask, so turning it off
                 _model_output = self.pre_model(input_ids=tokens['input_ids'],
-                                               # attention_mask=tokens['attention_mask'],
+                                                attention_mask=tokens['attention_mask'],
```

Comment:
Just note that transformer models like gerpt2 typically perform worse with left-side attention mask, so turning it off is generally advised. On the server-side retokenization there is padding introduced for tokenizers not equivalent to std_tokenizer.

Reply:
Nice catch Taco! That wasnt suppose to be there. I testing different parts of the model